.rultor.yml

```diff
@@ -9,17 +8,34 @@ assets:
   secring.gpg: yegor256/polystat-secrets#secring.gpg
   docker-password: yegor256/polystat-secrets#docker-password
 install: |
-  cp ../gradle.properties .
-  wget https://www.antlr.org/download/antlr-4.9.2-complete.jar
+  pdd --file=/dev/null
 merge:
   script: |
-    gradle build --info
+    mvn clean install --errors -Dstyle.color=never
 release:
   script: |-
     [[ "${tag}" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]] || exit -1
-    gradle build fatJar publish -x test --info -Dcandidates=false -PmvnPublicationVersion=${tag} -Dsigning.secretKeyRingFile=/home/r/secring.gpg
-    mv "J2EO-${tag}.jar" j2eo.jar
     image=yegor256/j2eo
     sudo docker build "$(pwd)" --tag "${image}:${tag}"
     cat ../docker-password | sudo docker login --password-stdin --username yegor256
-    sudo docker push "${image}:${tag}"
\ No newline at end of file
+    sudo docker push "${image}:${tag}"
```

Comment:
@includealex what exactly do you push to docker? you don't have `.jar` at this point of time